Makefile

```diff
@@ -378,6 +378,9 @@ go-agent-build-no-cgo: $(BUILD_AGENT_TARGETS)
 .PHONY: go-client-build
 go-client-build: $(BUILD_CLIENT_TARGETS)
 
+.PHONY: go-client-build
+go-client-build: $(BUILD_CLIENT_TARGETS)
+
```

Comment:
this is duplicated, remove

Reply:
Actually, looks like it got duplicated in 3.2 as well, so I'll remove it there. Leave this to avoid conflicts

Reply:
Oops